docs/integrations/sigrid-api-documentation.md

```diff
@@ -48,15 +48,6 @@ Sigrid's REST API mimics this behavior, as follows:
 
 ## Available end points
 
-* [Maintainability ratings](#maintainability-ratings)
-* [Security and reliability findings](#security-and-reliability-findings)
-* [Vulnerable libraries in Open Source Health](#vulnerable-libraries-in-open-source-health)
-* [Architecture Quality ratings](#architecture-quality-ratings)
-* [Architecture Quality data](#architecture-quality-data)
-* [System metadata](#system-metadata)
-* [System lifecycle management](#system-lifecycle-management)
-* [System and portfolio objectives](#system-objectives)
-
```

Comment:
I'll miss this list (unless there's an auto-generated ToC).

Reply:
There is! The `<sig-toc>` thingy automatically creates it based on the header tags in the HTML.